src/Main.java

```diff
@@ -1,5 +1,68 @@
+import java.util.Arrays;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Задача 1");
+
+        int[] arrInt = new int[]{1, 2, 3};
```

Comment:
этот способ создания массива аналогичн такому
```suggestion
        int[] arrInt = {1, 2, 3};
```
по идее требовалось создать массив указанной длины и затем заполнить его значениями, как показано в подсказке